docs/_themes/clusterhq/layout_docs.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>{{ title|striptags|e }}{{ titlesuffix }}</title>
+
+    <link href="{{ pathto('_static/css/bootstrap.min.css', 1) }}" rel="stylesheet">
+    <link href="{{ pathto('_static/css/' + style, 1) }}" rel="stylesheet">
+    <link href="{{ pathto('_static/css/pygments.css', 1) }}" rel="stylesheet">
+    
+    <script type="text/javascript" src="//fast.fonts.net/jsapi/b20e6cd5-c1a4-4878-9d69-da3806cb86b8.js"></script>
```

Comment:
Hm.  I'm not sure what the point of loading things over HTTPS when the page itself (or any of the content, really) is loaded over HTTP.  Little or no extra protection is granted but caching is worsened and page load times are slightly slowed.  However, I don't think those things are worth making changes for.  So if this isn't going to give anyone mixed content warnings now, great, scratch this comment.